deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,285 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "pmem-csi.intel.com/v1alpha1",
+          "kind": "Deployment",
+          "metadata": {
+            "name": "pmem-csi.intel.com"
+          },
+          "spec": {
+            "deviceMode": "lvm",
+            "nodeSelector": {
+              "storage": "pmem"
+            },
+            "pmemPercentage": 50
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Storage
+    certified: "false"
+    containerImage: intel/pmem-csi-driver:v0.7.0
+    createdAt: 2020-06-25 10:50Z
+    description: An operator for deploying and managing the PMEM-CSI driver
+    repository: https://github.com/intel/pmem-csi/tree/release-0.7
+    support: Intel
+  name: pmem-csi-operator.v0.7.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Deployment
+      name: deployments.pmem-csi.intel.com
+      resources:
+      - kind: StatefulSet
+        name: ""
+        version: apps/v1
+      - kind: DaemonSet
+        name: ""
+        version: apps/v1
+      - kind: Secret
+        name: ""
+        version: v1
+      - kind: Service
+        name: ""
+        version: v1
+      specDescriptors:
+      - description: 'CSI Driver mode for device management: ''lvl'' or ''direct'''
+        displayName: Device manager to use
+        path: deviceMode
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      version: v1alpha1
+  description: |2
+
+    Intel PMEM-CSI is a [CSI](https://github.com/container-storage-interface/spec)
+    storage driver for container orchestrators like Kubernetes. It makes local
+    persistent memory ([PMEM](https://pmem.io/)) available as a filesystem volume to
+    container applications. It can currently utilize non-volatile memory devices
+    that can be controlled via the [libndctl utility library](https://github.com/pmem/ndctl).
+    *Persistent memory* in this context refers to a non-volatile dual in-line
+    memory module (NVDIMM). See [PMEM-CSI documentation](http://githbu.com/intel/pmem-csi)
```

Comment:
Typo, https, missing article. We also should link to the docsite instead of our git repo.

=>
```
See the [PMEM-CSI documentation](https://intel.github.io/pmem-csi/0.7/README.html) for more details.
```

Reply:
Done

Reply:
Done.